pkg/parser/enrich.go

```diff
@@ -85,17 +86,27 @@ func GenDateParse(date string) (string, time.Time) {
 			return string(retstr), t
 		}
 	}
-	return retstr, time.Time{}
+
+	now := time.Now()
```

Comment:
You should check if the param date is empty or not. Because the retstr and the time returned are both overwritten.

Reply:
I see nothing overwritten here. If the param date is parsed, then it has already returned directly from the for loop. So if we are in this code, then date parsing failed.

btw I removed the retstr var because it wasn't scoped correctly.

Reply:
my bad, read it too fast.